vendor/bat-native-ledger/src/bat/ledger/internal/contribution/contribution.cc

```diff
@@ -58,7 +58,8 @@ void Contribution::Initialize() {
   for (const auto& value : currentReconciles) {
     ledger::CurrentReconcileProperties reconcile = value.second;
 
-    if (reconcile.retry_step == ledger::ContributionRetry::STEP_FINAL) {
+    if (reconcile.retry_step == ledger::ContributionRetry::STEP_FINAL ||
+        reconcile.retry_step == ledger::ContributionRetry::STEP_NO) {
```

Comment:
You can see that we ignore this step here https://github.com/brave/brave-core/blob/master/vendor/bat-native-ledger/src/bat/ledger/internal/contribution/contribution.cc#L595